theseus_gui/src-tauri/tauri.conf.json

```diff
@@ -13,8 +13,16 @@
   "tauri": {
     "allowlist": {
       "dialog": {
+        "confirm": true,
         "open": true
       },
+      "fs": {
+        "readFile": true,
+        "writeFile": true,
+        "exists": true,
+        "scope": ["*"]
+      },
```

Comment:
why does tauri need the FS permission? this should not be needed. all fs calls should be handled by the theseus library